### PR TITLE
sign in dialog refresh

### PR DESCRIPTION
**Pull request by eanders-MS**
Old:
image

New:
image

Later today I'll draft a blog post and point the "Learn More" link to it. I'm thinking we can detail how to create a Microsoft Account from a gmail address or phone number there.

### CODE REVIEW

**Comment by Jaqster** [comment]
Looks great!  I wonder if we should point the Learn More to a documentation page though instead of blog post?

**Comment by Jaqster** [comment]
Maybe add a "Sign In" page under Miscellaneous?  @ganicke can you help?
image

**Comment by eanders-MS** [comment]
Great idea! Should probably have one for Sign In and another for Cloud Sync. Will add them.

**Comment by eanders-MS** [comment]
@ganicke Is it possible to filter docs by target? We would want to only show these docs pages in Arcade, but would be great to also link to them from Arcade's table of contents.

**Comment by ganicke** [comment]
I guess the question is that you'd like them to go in as `common-docs` for use later with other targets but want to have it exposed now only for Arcade? It can go into the SUMMARY for Arcade but it could also be seen when someone inadvertently accesses `https://makecode.microbit.org/sign-in`. There is no per target doc filter. I think SUMMARY kind of is supposed to do that but it doesn't restrict the doc set.

We can just put it in Arcade docs and then move it down to pxt as an extended page when other targets will need it.

**Comment by eanders-MS** [comment]
Ok, sounds good! Where do you suggest I put the files? pxt-arcade/docs/static?

File: theme/common.less [hunk 1]
```diff
@@ -1915,33 +1915,8 @@ div.simframe.ui.embed {
...
-            justify-self: center;
-        }
-
-        div.providers {
-            display: grid;
-            row-gap: 20px;
-            grid-template-rows: auto;
-            justify-content: center;
-        }
+    div.remember-me {
```
Comment (livcheerful) [inline theme/common.less:1918]: should this be generalize CSS for the actions added to the modal?

**Comment by ganicke** [comment]
For any related doc pages, let's make a path under `./common-docs` such as:

https://github.com/microsoft/pxt/tree/master/common-docs/identity

-- or --

https://github.com/microsoft/pxt/tree/master/common-docs/accounts

This will allow a target to override the page if needs to include some additional info. We do that with extensible pages like https://github.com/microsoft/pxt/blob/master/common-docs/translate.md and some others.

**Comment by eanders-MS** [comment]
@ganicke I've added docs with some seed content. Would love a review, and please make changes as you see fit!

**Review by ganicke — APPROVED** [review]

File: theme/common.less (hunk 1 above)
Comment (eanders-MS) [reply theme/common.less]: > should this be generalized CSS for the actions added to the modal?

Good call. Done.

File: common-docs/identity/sign-in.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ Your signed in status is visible only to you. In the future we may add features
 
 ## Do I already have a Microsoft Account?
 
-It is possible! Do you play Minecraft, game on an Xbox, subscribe to Office 365, or use any other Microsoft products or services? If so, you probably already have a Microsoft Account.
+It is possible! Do you play Minecraft, games on an Xbox, subscribe to Office 365, or use any other Microsoft products or services? If so, you probably already have a Microsoft Account.
```
Comment (eanders-MS) [inline common-docs/identity/sign-in.md:27]: @ganicke I was using "game" as a verb here, making a loose affinity to gamer culture. "games" doesn't sound right to me.

File: common-docs/identity/sign-in.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+Your signed in status is visible only to you. In the future we may add features that allow others to see your presence. When that time comes, we would ask you to create a MakeCode Profile so that your name and picture are decoupled from your MSA profile.
+
+## Do I already have a Microsoft Account?
+
+It is possible! Do you play Minecraft, games on an Xbox, subscribe to Office 365, or use any other Microsoft products or services? If so, you probably already have a Microsoft Account.
+
+## Does this mean I have to create a new email address?
+
+No. You can create an MSA using your current email address if it is not hosted by Microsoft (e.g.: GMail, Yahoo Mail, and others). When creating your MSA account, just enter your existing email address.
```
Comment (livcheerful) [inline common-docs/identity/sign-in.md:31]: I think it should be styled as "Gmail" (at least that's what it looks like on their docs https://developers.google.com/gmail/design/css)

**Review by livcheerful — APPROVED** [review]

File: common-docs/identity/sign-in.md (hunk 2 above)
Comment (eanders-MS) [reply common-docs/identity/sign-in.md]: I went ahead and reverted this to "game". If you like we can reword it more generally, but in this context "game" felt right to me :).

Comment (ganicke) [reply common-docs/identity/sign-in.md]: I see. "game" or maybe "do gaming on Xbox".

**Pull request merged**